torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -742,6 +744,155 @@ void BuildProfilerSubmodule(py::module* m) {
       });
 }
 
+class PyLoweringContext {
+ public:
+  PyLoweringContext() : PyLoweringContext(GetCurrentDevice()) {}
+
+  PyLoweringContext(torch::lazy::BackendDevice device)
+      : lowering_ctx("PyLoweringContext", device) {}
+
+  // Builds a HLO graph given a set of output tensors. It is assumed that the
+  // tensors passed to this function are the result of the forward pass of a
+  // given Module/function.
```

Comment:
this is not necessary true right? builder just takes a list a tensor and treat them as root then do post-order-traversal.  You can pretty much pass any tensor to it.

Reply:
Yes, this comment is outdated, we can fix this.